contracts/modules/capital/Pool.sol

```diff
@@ -137,7 +140,14 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
         continue;
       }
 
-      total += getAssetValueInEth(assets[i].assetAddress);
+      address assetAddress = assets[i].assetAddress;
+      uint amountInSwapOperator;
```

Comment:
this amount in swap operator is asset specific. the name `amountInSwapOperator` makes me think its not, is it worth prefixing with `asset`?